apps/financial-aid/backend/src/app/lib/userContextExtractor.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  decodeToken,
+  IDENTITY_SERVER_SESSION_TOKEN_COOKIE_NAME,
+  User,
+} from '@island.is/financial-aid/shared/lib'
+import { ExecutionContext, UnauthorizedException } from '@nestjs/common'
+import { GqlExecutionContext } from '@nestjs/graphql'
+
+export const getUserFromContext = (
+  context: ExecutionContext & { contextType?: string },
+): User => {
+  const req =
+    context.contextType === 'graphql'
+      ? GqlExecutionContext.create(context).getContext().req
```

Comment:
I don't think this will ever happen. `financial-aid/backend` is implementing a REST server and won't have any inkling of GraphQL context.

Reply:
True, I was originally going to use the same decorator, but then it ended up not making sense, will remove this.